src/main/java/org/jenkinsci/plugins/pipeline/utility/steps/maven/ReadMavenPomStep.java

```diff
@@ -173,12 +191,33 @@ public boolean permitsStaticMethod(@Nonnull Method method, @Nonnull Object[] arg
 
         @Override
         public boolean permitsFieldGet(@Nonnull Field field, @Nonnull Object receiver) {
-            return receiver.getClass().getPackage().getName().equals(ORG_APACHE_MAVEN_MODEL);
+
+            if (receiver == null) {
```

Comment:
But nonnull is informative, we could receive a null right?
El 26 feb. 2016 6:51 p. m., "Jesse Glick" <notifications@github.com>
escribió:

> In
> src/main/java/org/jenkinsci/plugins/pipeline/utility/steps/maven/ReadMavenPomStep.java
> <https://github.com/jenkinsci/pipeline-utility-steps-plugin/pull/12#discussion_r54276812>
> :
>
> > @@ -173,12 +191,33 @@ public boolean permitsStaticMethod(@Nonnull Method method, @Nonnull Object[] arg
> >
> >          @Override
> >          public boolean permitsFieldGet(@Nonnull Field field, @Nonnull Object receiver) {
> > -            return receiver.getClass().getPackage().getName().equals(ORG_APACHE_MAVEN_MODEL);
> > +
> > +            if (receiver == null) {
>
> [image: :ant:] ditto
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jenkinsci/pipeline-utility-steps-plugin/pull/12/files#r54276812>
> .
>